CMakeLists.txt

```diff
@@ -144,6 +144,16 @@ if(NOT CLANG_RESOURCE_DIR)
     endif()
     set(tried "${tried}\n  ${d}/clang/${v}")
   endforeach()
+  if(NOT CLANG_RESOURCE_DIR)
+    foreach(d ${LLVM_LIBRARY_DIRS})
+      string(REGEX REPLACE  "llvm.*" "" dd ${d})
```

Comment:
This may replace too much.  Please use `get_filename_component` to compute the sibling directories.